package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mojaloop/ml-testing-toolkit-client-lib",
   "description": "Testing Toolkit Client Library",
-  "version": "0.0.6",
+  "version": "1.0.0-snapshot.0",
```

Comment:
You are missing `engines` in this package.json (e.g. https://github.com/mojaloop/central-ledger/blob/master/package.json#L26)

```
  "engines": {
    "node": "=16.x"
  },
```

Reply:
Done